client/lib/wp/browser.js

```diff
@@ -40,6 +40,11 @@ if ( config.isEnabled( 'oauth' ) ) {
 	} );
 }
 
+// set global wpcom instance
+if ( config.isEnabled( 'expose-global-wpcom' ) ) {
```

Comment:
Rather than setting a custom config value, do you think it might make sense to just test against the environment directly?

```js
if ( 'development' === config( 'env' ) ) {
	window.wpcom = wpcom;
}
```